server_addon/maya/server/settings/publishers.py

```diff
@@ -35,6 +35,76 @@ def angular_unit_enum():
+        },
+        {"label": "eulerFilter", "value": "eulerFilter"},
+        {"label": "noNormals", "value": "noNormals"},
+        {"label": "preRoll", "value": "preRoll"},
+        {"label": "renderableOnly", "value": "renderableOnly"},
+        {"label": "stripNamespaces", "value": "stripNamespaces"},
+        {"label": "uvWrite", "value": "uvWrite"},
+        {"label": "uvsOnly", "value": "uvsOnly"},
+        {"label": "verbose", "value": "verbose"},
+        {"label": "wholeFrameGeo", "value": "wholeFrameGeo"},
+        {"label": "worldSpace", "value": "worldSpace"},
+        {"label": "writeColorSets", "value": "writeColorSets"},
+        {"label": "writeFaceSets", "value": "writeFaceSets"},
+        {"label": "writeNormals", "value": "writeNormals"},
+        {"label": "writeUVSets", "value": "writeUVSets"},
+        {"label": "writeVisibility", "value": "writeVisibility"}
+    ]
+
+
+def extract_alembic_data_format_enum():
+    return [
+        {"label": "ogawa", "value": "ogawa"},
+        {"label": "HDF", "value": "HDF"}
+    ]
+
+
+def extract_alembic_overrides_enum():
+    return [
+        {"value": "attr", "label": "Custom Attributes"},
+        {"value": "attrPrefix", "label": "Custom Attributes Prefix"},
+        {"value": "autoSubd", "label": "autoSubd"},
+        {"value": "dataFormat", "label": "dataFormat"},
+        {
+            "value": "dontSkipUnwrittenFrames",
+            "label": "dontSkipUnwrittenFrames"
+        },
+        {"value": "eulerFilter", "label": "eulerFilter"},
+        {"value": "melPerFrameCallback", "label": "melPerFrameCallback"},
+        {"value": "melPostJobCallback", "label": "melPostJobCallback"},
+        {"value": "noNormals", "label": "noNormals"},
+        {"value": "preRoll", "label": "preRoll"},
+        {"value": "preRollStartFrame", "label": "Pre Roll Start Frame"},
+        {"value": "pythonPerFrameCallback", "label": "pythonPerFrameCallback"},
+        {"value": "pythonPostJobCallback", "label": "pythonPostJobCallback"},
+        {"value": "renderableOnly", "label": "renderableOnly"},
+        {"value": "stripNamespaces", "label": "stripNamespaces"},
+        {"value": "userAttr", "label": "userAttr"},
+        {"value": "userAttrPrefix", "label": "userAttrPrefix"},
+        {"value": "uvWrite", "label": "uvWrite"},
+        {"value": "uvsOnly", "label": "uvsOnly"},
+        {"value": "verbose", "label": "verbose"},
+        {"value": "visibleOnly", "label": "Visible Only"},
+        {"value": "wholeFrameGeo", "label": "wholeFrameGeo"},
+        {"value": "worldSpace", "label": "worldSpace"},
+        {"value": "writeColorSets", "label": "writeColorSets"},
+        {"value": "writeCreases", "label": "writeCreases"},
+        {"value": "writeFaceSets", "label": "writeFaceSets"},
+        {"value": "writeNormals", "label": "writeNormals"},
+        {"value": "writeUVSets", "label": "writeUVSets"},
+        {"value": "writeVisibility", "label": "writeVisibility"}
+    ]
```

Comment:
Is there a reason that Pre Roll Start Frame is the only one that has a 'nice' label?